crates/rolldown_common/src/inner_bundler_options/mod.rs

```diff
@@ -29,16 +38,23 @@ pub struct BundlerOptions {
   pub dir: Option<String>,
   pub format: Option<OutputFormat>,
```

Comment:
https://github.com/rolldown-rs/rolldown/blob/ee3739d65dfed173dbc582270f7cc89017dc285a/crates/rolldown_common/src/inner_bundler_options/types/output_format.rs

Missed this one

Reply:
done